corehq/apps/reports/standard/monitoring.py

```diff
@@ -1792,22 +1801,74 @@ def _total_row(self, rows, report_data):
                     if report_data.submissions_by_user.get(user['user_id'], False):
                         active_users.add(user['user_id'])
                     all_users.add(user['user_id'])
-            total_row[3] = '%s / %s' % (len(active_users), len(all_users))
+            total_row[3] = {'numerator': len(active_users), 'demoninator': len(all_users)}
         else:
-            total_row[3] = '%s / %s' % (num, len(rows))
+            total_row[3] = {'numerator': num, 'denominator': len(rows)}
         return total_row
 
+    @property
+    def get_all_rows(self):
+        user_es_query = EMWF.user_es_query(
+            self.domain, self.request.GET.getlist(EMWF.slug), self.request.couch_user
+        )
+        chunk_size = 10000
+        first_id_in_chunk = 0
+        user_chunk_total = user_es_query.fields(util.SimplifiedUserInfo.ES_FIELDS).size(chunk_size).run().total
+        while first_id_in_chunk < user_chunk_total:
+            user_chunk = user_es_query.fields(util.SimplifiedUserInfo.ES_FIELDS).start(first_id_in_chunk).size(chunk_size).run().hits
+            users = [util._report_user_dict(user) for user in user_chunk]
+            formatted_data = self._report_data(users_to_iterate=users)
+            if self.view_by_groups:
+                rows = self._rows_by_group(formatted_data)
```

Comment:
I don't think this'll work, since its unlikely all users in a group are going to be in a single chunk.  I'd try to make this feature unavailable for `view_by_groups`.

Reply:
This is addressed in this commit: https://github.com/dimagi/commcare-hq/pull/23782/commits/a2290d9a82232dacc981120eda4f1e2a1b0eaf9d